store/dbstore.go

```diff
@@ -197,3 +198,24 @@ func (dbw *DBStoreWrite) Close() error {
 func (dbw *DBStoreWrite) Commit() error {
 	return dbw.dbs.writeBytes(dbw.name, &dbw.data)
 }
+
+// Update - updates an existing dataentry, or creates a new one if none exists
```

Comment:
@pasinskim I added a couple of more methods to the database so that it will be easier to store say, structs etc. I know that @bboozzoo wants a text marshaller interface instead of a binary one, so that reading data out of the database manually is easier, but I think that a binary one is plenty as long as one is careful. Thoughts?